src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -7,7 +7,6 @@ class EnrollmentHistory extends React.Component {
   render() {
     const enrollmentData = this.props.enrollmentData || {};
     const enrollments = enrollmentData.enrollments || [];
-    const currentlyAllowed = enrollmentData.percentageBenefit !== 0 || enrollmentData.originalEntitlement !== 0;
 
     // TODO: find out when this warning should be shown.
```

Comment:
This expiration warning no longer appears in any mocks, so I think we can delete all this code.